pkg/scheduler/cache/cache.go

```diff
@@ -482,6 +482,8 @@ func (sc *SchedulerCache) processCleanupJob() {
 		return
 	}
 
+	defer sc.deletedJobs.Done(obj)
```

Comment:
If there is something error in line 496, we'll try to delete it in line 501. If we defer delete it, it will be deleted at last, line 501 will take no effort.

Reply:
`deleteJob` will add it to `deletedJob` queue, as `Done` is not executed, it's only marked as dirty in queue :)

Reply:
oh, there's still an issue that after `delete(sc.Jobs, job.UID)`, we should forget job in reateLimitedQueue.

@hzxuzhonghu

Reply:
The forget  method for `DefaultControllerRateLimiter` is empty, have no effect. 

```
func (r *BucketRateLimiter) Forget(item interface{}) {
}

```

Reply:
It's better to include to align with its description in interface.